src/process/process_set.cu

```diff
@@ -4,71 +4,92 @@
 
 #include <chrono>
 #include <iostream>
+#include <micm/util/cuda_param.hpp>
+
+//device pointers passing to AddForcingTermsKernel()
+typedef struct forcingDevice{
+  double* rate_constants; 
+  double* state_variables; 
+  double* forcing; 
+  size_t* number_of_reactants; 
+  size_t* reactant_ids; 
+  size_t* number_of_products; 
+  size_t* product_ids; 
+  double* yields; 
+};
+//device pointers passing to AddJacobianTermsKernel() 
+typedef struct jacobianDevice{
+  double* rate_constants; 
+  double* state_variables; 
+  double* jacobian;
+  size_t* number_of_reactants; 
+  size_t* reactant_ids; 
+  size_t* number_of_products; 
+  double* yields; 
+  size_t* jacobian_flat_ids; 
+};
+
+const size_t BLOCK_SIZE = 320;
```

Comment:
is this a constant that applies to all the CUDA code? if so, should it be defined in a common location?